app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb

```diff
@@ -1075,7 +1075,7 @@ def self.vm_inv_to_disk_hashes(inv, storage_uids, storage_profile_by_disk_mor =
           new_result = {
             :device_name     => device.fetch_path('deviceInfo', 'label'),
             :device_type     => device_type,
-            :controller_type => controller_type,
+            :controller_type => %w(floppy cdrom-raw cdrom-image).include?(device_type) ? vm_mor : controller_type,
```

Comment:
I do not need it. What I need it is to make drives unique and it appears that for floppy and cdrom are not for vmware and regular drives for openstack.